src/nvim/tag.c

```diff
@@ -1411,7 +1411,7 @@ int find_tags(char_u *pat, int *num_matches, char ***matchesp, int flags, int mi
   int matchoff = 0;
   int save_emsg_off;
 
-  char_u *mfp;
+  char *mfp;
```

Comment:
This looks strange as there are a lot of `sizeof(char_u)` and `sizeof(char_u *)` below.

Reply:
Aren't they the same size anyway?

I can change the sizeof types.